app/assets/stylesheets/_vocab_sheet.scss

```diff
@@ -250,6 +193,97 @@
       z-index: 11;
     }
   }
+
+  &__bar {
+    @include border-radius-small;
+    width: 10rem;
+    position: absolute;
+    right: 1rem;
+    z-index: 100;
+    background-color: $vocab-bg;
+    overflow: hidden;
+    height: 90vh;
+    display: none;
+
+    &:after {
+      content: '';
+      position: absolute;
+      z-index: 2;
+      bottom: 0;
+      left: 0;
+      width: 100%;
+      height: 3rem;
+      background: linear-gradient(transparent, $vocab-bg);
+    }
+
+    &-header {
```

Comment:
would it be a lot of work to convert this and the other ones below from "-header" to element naming "__header"?

Reply:
oh wait this is already nested inside an element. ignore me